client/me/privacy/main.jsx

```diff
@@ -163,7 +163,7 @@ const Privacy = createReactClass( {
 					</form>
 				</Card>
 				<SectionHeader
-					label={ translate( 'Data Processing Addendum', {
+					label={ translate( 'Data processing addendum', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.